### ISSUE TITLE
Feature request: Add option to "Open.." a file to newman native app instead of just importing it

### ISSUE THREAD

**Issue by gaieges** [a85/POSTMan-Chrome-Extension#3753]
Feature request: be able to edit a collection file on disk on the fly, instead of having to import and export the collection as desired.

The case that I run into that is an annoyance from time to time is during editing of a collection file in git that gets used downstream in my CI pipeline. In its current form, I'm required to import the existing collection from git, and then export it to the file system in it's place once I'm done.

It'd be great instead to just be able to open a file and save it directly to where I opened it from, and then git commit and have my CI pipeline use it.

**Comment by prashantagarwal** [comment]
@gaieges I am not sure about this. You can anyways edit you collection file using some IDE, it is just that you must be aware what a property inside collection JSON represent.

**Comment by gaieges** [comment]
Yeah I edit collection files frequently via IDE's / text editors, thats why I've put in this ticket because it's really cumbersome to do so (the json syntax specifically, I have another ticket open to request supporting YAML files as well for ease of editing).

**Comment by shashankawasthi88** [comment]
Hi @gaieges ,

As of [Postman v12](https://www.postman.com/downloads/), we now support Native Git Integration.
You can connect a workspace directly to a local Git repository and work against your filesystem. Collections and related artifacts are stored as files, enabling standard Git workflows (branching, diffs, PRs, CI/CD) without manual export/import.
Docs: https://learning.postman.com/docs/agent-mode/native-git
Announcement: https://blog.postman.com/new-postman-is-here/
Feel free to try it out and share feedback.

https://github.com/user-attachments/assets/fdf600d7-5285-4e34-83e4-5d2d7794c2ed
